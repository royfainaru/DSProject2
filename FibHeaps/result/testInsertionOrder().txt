╮
├── ╭────────────────╮
│   │ Key: 0         │
│   ├────────────────┤
│   │ Rank: 0        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 2        │
│   │ Prev: (null)   │
│   │ Child: (null)  │
│   ╰────────────────╯
├── ╭────────────────╮
│   │ Key: 2         │
│   ├────────────────┤
│   │ Rank: 0        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 8        │
│   │ Prev: 0        │
│   │ Child: (null)  │
│   ╰────────────────╯
├── ╭────────────────╮
│   │ Key: 8         │
│   ├────────────────┤
│   │ Rank: 0        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 5        │
│   │ Prev: 2        │
│   │ Child: (null)  │
│   ╰────────────────╯
╰── ╭────────────────╮
    │ Key: 5         │
    ├────────────────┤
    │ Rank: 0        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: (null)   │
    │ Prev: 8        │
    │ Child: (null)  │
    ╰────────────────╯
