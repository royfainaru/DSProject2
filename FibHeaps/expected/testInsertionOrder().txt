╮
├── ╭────────────────╮
│   │ Key: 0         │
│   ├────────────────┤
│   │ Rank: 0        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 2        │
│   │ Prev: 5        │
│   │ Child: (null)  │
│   ╰────────────────╯
├── ╭────────────────╮
│   │ Key: 2         │
│   ├────────────────┤
│   │ Rank: 0        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 8        │
│   │ Prev: 0        │
│   │ Child: (null)  │
│   ╰────────────────╯
├── ╭────────────────╮
│   │ Key: 8         │
│   ├────────────────┤
│   │ Rank: 0        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 5        │
│   │ Prev: 2        │
│   │ Child: (null)  │
│   ╰────────────────╯
╰── ╭────────────────╮
    │ Key: 5         │
    ├────────────────┤
    │ Rank: 0        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: 0        │
    │ Prev: 8        │
    │ Child: (null)  │
    ╰────────────────╯
